playbooks/group_vars/activemq.yml

```diff
@@ -0,0 +1,5 @@
+activemq_v: "{{ acs_play_activemq_version }}"
```

Comment:
why don't use `acs_play_activemq_version` straight everywhere you are using now `activemq_v`?